SparkPost/2web2ui
# FE-788 - Cypress integration tests for the Health Score feature

[FE-788](https://jira.int.messagesystems.com/browse/FE-788)

### What Changed
- Adds Cypress testing for the Health Score page and the Health Score by Subaccount page
- Adds fixtures for various Signals Analytics APIs
- Adds relevant `data-id` attributes for testing
- Prettier reformatting for several health score-related components
- Adds Signals Analytics tests to spec group for Travis CI runs

### How To Test
- Visit `/signals/health-score` and verify that these tests capture relevant behavior for the health score and health score by subaccount pages

### To Do
